tests/emodulus_calibration/Wolf 8.2 Probe 1.csv

```diff
@@ -0,0 +1,529 @@
+Force [kN],Transducer 1[mm],Transducer 2[mm],Transducer 3[mm]
```

Comment:
why is that csv file stored?

Reply:
We already discussed this, just for the test as in the dodo it is generated on the fly and this is nor present by default.